src/views/partials/events/SignupModal.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <button type="button" class="btn bg-emerald-500 text-white font-bold py-2 px-4 ml-2 rounded" @click="toggleModal()">
+    <i class="fas fa-user-plus mr-2"></i>Sign up
+  </button>
+
+  <div v-show="isOpen" class="absolute inset-0 flex items-center justify-center bg-black-deep bg-opacity-70">
+    <div class="relative bg-white rounded-lg shadow dark:bg-black-light w-1/3">
+      <div class="flex justify-between items-start p-4 rounded-t border-b dark:border-gray-600">
+        <h3 class="text-xl font-semibold text-gray-900 dark:text-white m-0">Sign Up</h3>
+        <button
+          type="button"
+          class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center dark:hover:bg-gray-600 dark:hover:text-white"
+          @click="toggleModal()"
+        >
+          <svg
+            aria-hidden="true"
+            class="w-5 h-5"
+            fill="currentColor"
+            viewBox="0 0 20 20"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              fill-rule="evenodd"
```

Comment:
Might be cleaner to use font awesome as it's already loaded?

Reply:
Fixed in [b401cbc]